plugins/vpc-eni/network/network_windows.go

```diff
@@ -395,7 +395,7 @@ func (nb *NetBuilder) disableInterface(adapterName string) error {
 		// Disable the interface using netsh.
 		log.Infof("Disabling management vNIC %s in the host namespace.", adapterName)
 		commandString := fmt.Sprintf(netshDisableInterface, adapterName)
-		cmd := exec.Command("cmd", "/C", commandString)
+		cmd := exec.Command("powershell", "-C", commandString)
```

Comment:
I will merge this because "netsh interface" context is deprecated, and using PS cmdlets is a better way to interact with network interfaces anyway. But there really shouldn't be a difference between disabling an interface in netsh vs PS. What you are seeing has probably a good explanation.